cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -963,6 +963,17 @@ body {
                 padding-right: 0;
             }
 
+            &-item {
+                // Move the first link in each mega menu list group up a bit so that it aligns
+                // with the featured links box. We do this instead of removing the top padding
+                // to preserve the links' hover state.
+                &:first-child {
+                    a {
```

Comment:
Since we have a named class for links, let's target that instead of an anonymous `a`. How about this instead for this whole block?

```less
            // Move the first link in each mega menu list group up a bit so that it aligns
            // with the featured links box. We do this instead of removing the top padding
            // to preserve the links' hover state.
            &-item:first-child &-link {
                margin-top: unit( -12px / @base-font-size-px, em );
            }
```